docs/airgap.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Air-Gapped Installation
+title: ''
+---
+
+## Install Elemental in an Air-Gapped Environment
+
+### Assumptions
+A Rancher air-gapped installation should be already configured as per the [official Rancher documentation](https://ranchermanager.docs.rancher.com/pages-for-subheaders/air-gapped-helm-cli-install).
+In particular a local registry should be available in the air-gapped infrastructure.
+
+:::info authenticated access to the private registry is not supported yet
+At this stage Elemental does not support installation from an authenticated registry: the private registry should allow unauthenticated pulls from inside the airgap environment.
```

Comment:
Is it lacking in Elemental only or also in Rancher ?

Reply:
not sure about Rancher, I will check